src/csl/mod.rs

```diff
@@ -1579,7 +1597,7 @@ impl<'a> StyleContext<'a> {
 
     /// Check whether to do punctuation in quotes.
     fn punctuation_in_quotes(&self) -> bool {
-        self.lookup_locale(|f| f.style_options?.punctuation_in_quote)
```

Comment:
Don't we need to propagate the entry language to retrieve the right setting?

Reply:
I have a few experience with international bibliography standarts, so I don't really understand which things may change with different language. My goal was to achieve correct translation of words when non-english language is used. I would appreciate any additional information.

Reply:
I think... I'll propogate it.